src/chaospy/dist/baseclass.py

```diff
@@ -416,31 +416,31 @@ def __div__(self, X):
 
     def __rdiv__(self, X):
         """Y.__rdiv__(X) <==> X/Y"""
-        return chaospy.dist.operators.mul(X, self**-1)
+        return chaospy.dist.operators.mul(self, X**-1)
 
     def __floordiv__(self, X):
         """Y.__floordiv__(X) <==> Y/X"""
         return chaospy.dist.operators.mul(self, X**-1)
 
     def __rfloordiv__(self, X):
         """Y.__rfloordiv__(X) <==> X/Y"""
-        return chaospy.dist.operators.mul(X, self**-1)
+        return chaospy.dist.operators.mul(self, X**-1)
 
     def __truediv__(self, X):
         """Y.__truediv__(X) <==> Y/X"""
         return chaospy.dist.operators.mul(self, X**-1)
 
     def __rtruediv__(self, X):
         """Y.__rtruediv__(X) <==> X/Y"""
-        return chaospy.dist.operators.mul(X, self**-1)
+        return chaospy.dist.operators.mul(self, X**-1)
 
     def __pow__(self, X):
         """Y.__pow__(X) <==> Y**X"""
         return chaospy.dist.operators.pow(self, X)
 
     def __rpow__(self, X):
         """Y.__pow__(X) <==> X**Y"""
-        return chaospy.dist.operators.pow(X, self)
+        return chaospy.dist.operators.pow(self, X)
```

Comment:
Following the text above, these seems incorrect.

The doc in `__rpow__` is wrong. Feel free to fix it.